projects/Allwinner/bootloader/update.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
+
+[ -z "$SYSTEM_ROOT" ] && SYSTEM_ROOT=""
```

Comment:
What is the point of this because `SYSTEM_ROOT ` can be empty before and after.
If you want to check for unset variable use `${SYSTEM_ROOT+x}`. But I think it doesn't matter in usage.

Reply:
TBH this file was just copied and adjusted. Maybe @lrusak knows details.

Reply:
CORRECTION: we still need to keep the `$SYSTEM_ROOT` references in the `update.sh` scripts, this would normally be set to `sysroot` when called from `init`, but yeah, the single line pointed out by @vpeter4 which sets nothing to nothing is redundant, although harmless.

Up to you if you leave it in or not - it can be cleaned up in a later PR along with all the other `update.sh` scripts that have this same redundant code.